assets/multiqc_config.yml

```diff
@@ -261,5 +265,27 @@ custom_data:
     section_name: "Copy Number Signatures Activities"
     description: "The image is a summary view of Copy number Signatures from segmentation files, computed by the R Package CINSignatureQuantification."
 
+  hrdcna_summary:
+    id: "hrdcna"
+    name: "HRDCNA Summary"
+    section_name: "HRDCNA Summary"
+    format: "txt"
+    plot_type: "table"
+    description: "This table reports HRDCNAScores with the putative labels for the classification of the sample into 'HRD' or 'HRP'."
```

Comment:
Use HRD+ (HRD) and HRD- (HRP) perhaps, I was recently criticized for using HRD/HRP.